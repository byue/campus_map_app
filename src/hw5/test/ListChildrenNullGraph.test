# Tests listing children of null graph

# list children of null graph
ListChildren g1 n1
