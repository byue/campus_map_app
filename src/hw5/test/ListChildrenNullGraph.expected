# Tests listing children of null graph

# list children of null graph
g1 has not been created